core/src/test/scala/org/apache/spark/benchmark/BenchmarkBase.scala

```diff
@@ -49,7 +49,11 @@ abstract class BenchmarkBase {
       val resultFileName =
         s"${this.getClass.getSimpleName.replace("$", "")}$jdkString$suffix-results.txt"
       val prefix = Benchmarks.currentProjectRoot.map(_ + "/").getOrElse("")
-      val file = new File(s"${prefix}benchmarks/$resultFileName")
+      val dir = new File(s"${prefix}benchmarks/")
+      if (!dir.exists()) {
+        dir.mkdirs()
+      }
+      val file = new File(s"${dir}$resultFileName")
```

Comment:
Ah, okay. the new benchmark were added at SPARK-33882 and SPARK-35150, that was after https://github.com/apache/spark/pull/32015 and https://github.com/apache/spark/pull/32044.

Okay, @byungsoo-oh, would you mind throwing an exception with `assert` here with a pretty error message? I think the results files should always be existent. I think the results files at SPARK-33882 and SPARK-35150 were not mistakenly added.